core/src/main/java/com/ttbmp/cinehub/core/service/movie/MovieApiService.java

```diff
@@ -0,0 +1,21 @@
+package com.ttbmp.cinehub.core.service.movie;
+
+import com.ttbmp.cinehub.core.dto.MovieDto;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.List;
+
+/**
+ * @author Palmieri Ivan
+ */
+public interface MovieApiService {
```

Comment:
`connect` and `setMovie` are operations needed to the particular service that we're currently using. The interface should be as generic as possible,